ClientSide(FrontEnd)/clay_/src/components/forgot.css

```diff
@@ -0,0 +1,87 @@
+.start{
+    display: flex;
+    margin-top: -2%;
+    justify-content: center;
+    border-radius: 70px;
+}
+.start1{
+    padding: 30px;
+    width: 550px;
+    margin-top: -490px;
+    border-radius: 70px;
+    background-color: #FFB5C3;
+    background-image: url('/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/Screenshot from 2023-06-06 16-50-35.png');
+}
+.starting2{
+    padding: 30px;
+    width: 400px;
+    height: 350px;
+    border-radius: 70px;
+}
+.ame{
+    margin: 20px;
+    padding: 15px 40px;
+    margin-left: 60px;
+    border-radius: 30px;
+    background-color: white;
+    border: 2px solid white;
+
+}
+.sername{
```

Comment:
write correct spelling in the className

Reply:
that is a unwanted classname i  removed it